src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8295.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue8295 : _IssuesUITest
+	{
+		const string ToggleGlyphButton = "ToggleGlyphBtn";
+		public Issue8295(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Can't Change ToolbarItem FontIconSource Glyph After Load";
+
+		[Test]
+		[Category(UITestCategories.ToolbarItem)]
+		public void ToolbarItemFontIconSourceChangesAtRunTime()
+		{
+			App.WaitForElement(ToggleGlyphButton);
+			App.Tap(ToggleGlyphButton);
+			VerifyScreenshot();
```

Comment:
Pending snapshots. Are available in the latest build. Could you commit the images?
![image](https://github.com/user-attachments/assets/f8206dc5-4518-40d0-87a2-4c16a7286953)

Reply:
@jsuarezruiz , The Mac platform snapshot is not proper. I have attached the snapshots for remaining platforms. please let me know if you have any concerns.

**Mac**
![ToolbarItemFontIconSourceChangesAtRunTime (1)](https://github.com/user-attachments/assets/453d9f47-a9e6-408c-ab52-c2547cefdf57)

Reply:
Let me take a look.

Reply:
Will be fixed by https://github.com/dotnet/maui/pull/27492
Let's wait until merge it, then we can rebase and trigger a new build

Reply:
@jsuarezruiz , I have attached the pending Mac snapshot. Please let me know if you have any concerns.